airflow/jobs/scheduler_job.py

```diff
@@ -29,7 +29,7 @@
 from datetime import timedelta
 from typing import List, Set
 
-from setproctitle import setproctitle
+from setproctitle import setproctitle  # pylint: disable=no-name-in-module
```

Comment:
http://pylint.pycqa.org/en/latest/technical_reference/c_extensions.html

We could add setproctitle to to the list, or disable this warning globally  c-extension-no-member?

Also perhaps ignored-modules=setproctitle may be needed

Reply:
I am ok with adding this as exception in pylintrc